demos/python_demos/clip/client.py

```diff
@@ -0,0 +1,83 @@
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+import sys
+sys.path.append("../../common/python")
+import tritonclient.grpc as grpcclient
+import argparse
+import datetime
+import numpy as np
+from client_utils import print_statistics
+from urllib.request import urlretrieve
+from pathlib import Path
+import os
+from PIL import Image
+
+parser = argparse.ArgumentParser(description='Client for clip example')
+
+parser.add_argument('--url', required=False, default='localhost:9000',
+                    help='Specify url to grpc service. default:localhost:9000')
+parser.add_argument('--input_labels', required=False, default="cat,dog,wolf,tiger,man,horse,frog,tree,house,computer",
+                    help="Specify input_labels to the CLIP model. default:cat,dog,wolf,tiger,man,horse,frog,tree,house,computer")
+parser.add_argument('--image_url', required=False, default='https://storage.openvinotoolkit.org/repositories/openvino_notebooks/data/data/image/coco.jpg',
+                    help='Specify image_url to send to the CLIP model. default:https://storage.openvinotoolkit.org/repositories/openvino_notebooks/data/data/image/coco.jpg')
+parser.add_argument('--iterations', default=1,
+                        help='Number of requests iterations, as default use number of images in numpy memmap. default: 1 ',
+                        dest='iterations', type=int)
+args = vars(parser.parse_args())
+
+iterations = args.get('iterations')
+iteration = 0
+
+client = grpcclient.InferenceServerClient(args['url'])
+
+
+image_url = args['image_url']
+print(f"Using image_url:\n{image_url}\n")
+
+input_name = image_url.split("/")[-1]
+sample_path = Path(os.path.join("data", input_name))
+if not os.path.exists(sample_path):
+    sample_path.parent.mkdir(parents=True, exist_ok=True)
+    urlretrieve(
+        image_url,
+        sample_path,
+    )
+#image = Image.open(sample_path)
+with open(sample_path, "rb") as f:
+    data = f.read()
+
+infer_input = grpcclient.InferInput("image", [len(data)], "UINT8")
+infer_input._raw_content = data
+
+input_labels = args['input_labels'].split(",")
+print(f"Using input_labels:\n{input_labels}\n")
+labels_npy = np.array(input_labels)
+infer_input1 = grpcclient.InferInput("input_labels", [len(labels_npy)], labels_npy.dtype.str)
+infer_input1._raw_content = labels_npy.tobytes()
+
+processing_times = np.zeros((0),int)
+while iteration < iterations:
+    iteration += 1
+    print(f"Iteration {iteration}")
+    start_time = datetime.datetime.now()
+    results = client.infer("python_model", [infer_input, infer_input1])
+    end_time = datetime.datetime.now()
+    duration = (end_time - start_time).total_seconds() * 1000
+    processing_times = np.append(processing_times,np.array([int(duration)]))
+    print(f"Detection:\n{results.as_numpy('detection').tobytes().decode()}\n")
+
+batch_size = 1
```

Comment:
we only use it here right? can we remove it and in the line below go with:
`print_statistics(processing_times, batch_size=1)`